FinSearchUnified/Helper/StaticHelper.php

```diff
@@ -856,4 +852,14 @@ public static function setSmartDidYouMean(SimpleXMLElement $xmlResponse)
             ]);
         }
     }
+
+    public static function removeSpecialCharacters($name, $replace = '')
```

Comment:
Check my previous comment regarding this method. The use of it shouldn't be necessary unless there is an obvious reason that I'm missing.

Reply:
Please see my response, thanks.